doc/nrf/security/crypto/crypto_supported_features.rst

```diff
@@ -8359,6 +8367,14 @@ The options are grouped by Series and drivers available for the device Series, a
                  - Experimental
                  - --
                  - Experimental
+               * - WPA3-SAE
+                  - | :kconfig:option:`CONFIG_PSA_WANT_ALG_WPA3_SAE`
+                    | :kconfig:option:`CONFIG_PSA_WANT_ALG_WPA3_SAE_H2E`
+                  - Experimental
+                  - Experimental
+                  - Experimental
+                  - Experimental
+                  - Experimental
 
          .. tab:: nrf_oberon
```

Comment:
Also update nrf_oberon section. Both CRACEN and nrf_oberon tables need to be kept in sync as to the number of entries.